lib/nfs_common.pm

```diff
@@ -288,7 +288,7 @@ sub check_y2_nfs_func {
         systemctl('is-active nfs-server');
         systemctl('restart rpcbind');
         systemctl('is-active rpcbind');
-        systemctl('restart rpcbind.socket');
+        systemctl('restart rpcbind.socket', timeout =>120);
```

Comment:
120s are enough?

Reply:
> 120s are enough?
https://openqa.nue.suse.com/t7252337 PASSED
https://openqa.nue.suse.com/t7252338 PASSED
https://openqa.nue.suse.com/t7252339 PASSED

Reply:
You'd better to analyze the log to get the real needed time, so that we can know whether 120s is enough or not. Please update this in ticket.

Reply:
updated.

Reply:
Ok, I think 95s -> 120s is enough.